custom/polio_rdc/ucr/data_sources/users.json

```diff
@@ -0,0 +1,291 @@
+                    "type": "property_path"
+                },
+                "is_nullable": true,
+                "is_primary_key": false,
+                "transform": {},
+                "type": "expression"
+            },
+            {
+                "column_id": "email",
+                "comment": null,
+                "create_index": false,
+                "datatype": "string",
+                "display_name": "email",
+                "expression": {
+                    "datatype": null,
+                    "property_path": [
+                        "email"
+                    ],
+                    "type": "property_path"
+                },
+                "is_nullable": true,
+                "is_primary_key": false,
+                "transform": {},
+                "type": "expression"
+            },
+            {
+                "column_id": "language",
+                "comment": null,
+                "create_index": false,
+                "datatype": "string",
+                "display_name": "language",
+                "expression": {
+                    "datatype": null,
+                    "property_path": [
+                        "language"
+                    ],
+                    "type": "property_path"
+                },
+                "is_nullable": true,
+                "is_primary_key": false,
+                "transform": {},
+                "type": "expression"
+            },
+            {
+                "column_id": "last_sync",
+                "comment": null,
+                "create_index": false,
+                "datatype": "datetime",
+                "display_name": "last_sync",
+                "expression": {
+                    "datatype": null,
+                    "property_path": [
+                        "reporting_metadata",
+                        "last_sync_for_user",
+                        "sync_date"
+                    ],
+                    "type": "property_path"
+                },
+                "type": "expression"
+            },
+            {
+                "column_id": "last_app_version",
+                "comment": null,
+                "create_index": false,
+                "datatype": "string",
+                "display_name": "last_app_version",
+                "expression": {
+                    "datatype": null,
+                    "property_path": [
+                        "reporting_metadata",
+                        "last_build_for_user",
+                        "build_version"
+                    ],
+                    "type": "property_path"
+                },
+                "is_nullable": true,
+                "is_primary_key": false,
+                "transform": {},
+                "type": "expression"
+            },
+            {
+                "column_id": "is_active",
+                "comment": null,
+                "create_index": false,
+                "datatype": "small_integer",
```

Comment:
is there no bool type?

Reply:
it's quite odd. there is, but it throws a vaildation error if you try to use it. I didn't dig in further. https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/userreports/datatypes.py#L9